scripts/build/builders/android.py

```diff
@@ -301,20 +308,35 @@ def gradlewBuildExampleAndroid(self):
                         self.identifier, module),
                 )
         else:
-            self._Execute(
-                [
-                    "%s/examples/%s/android/App/gradlew"
-                    % (self.root, self.app.ExampleName()),
-                    "-p",
-                    "%s/examples/%s/android/App/" % (self.root,
-                                                     self.app.ExampleName()),
-                    "-PmatterBuildSrcDir=%s" % self.output_dir,
-                    "-PmatterSdkSourceBuild=false",
-                    "-PbuildDir=%s" % self.output_dir,
-                    "assembleDebug",
-                ],
-                title="Building Example " + self.identifier,
-            )
+            if self.app.ExampleName() == "st-device-app":
```

Comment:
use `elif` to not increase indent as much.

What is the difference here? why is using `-PbuildDir` not ok for st-app? A comment would help as branching out means we have more things non-similar which is harder to maitain.

Reply:
We need to add "-PbuildDir" and Also we can use "elif"